task.ps1

```
@@ -6,10 +6,14 @@ $subnetName = "default"
 $vnetAddressPrefix = "10.0.0.0/16"
 $subnetAddressPrefix = "10.0.0.0/24"
 $sshKeyName = "linuxboxsshkey"
-$sshKeyPublicKey = Get-Content "~/.ssh/id_rsa.pub" 
-$vmName = "matebox"
+$sshKeyPublicKey = Get-Content "~/.ssh/id_rsa.pub"
```

Reading the SSH public key from `Get-Content "~/.ssh/id_rsa.pub"` is brittle (shell tilde expansion and platform differences). Use a robust path and check for existence, for example:
- `$pubKeyPath = Join-Path $HOME '.ssh/id_rsa.pub'`
- `if (-not (Test-Path $pubKeyPath)) { Write-Error "Public key not found: $pubKeyPath"; exit 1 }`
- `$sshKeyPublicKey = Get-Content $pubKeyPath`
This prevents silent failures when the file is missing or path expansion behaves differently.